src/bucket.ts

```diff
@@ -1393,38 +1487,42 @@ class Bucket extends ServiceObject {
    *   const apiResponse = data[1];
    * });
    */
-  get(options, callback?) {
-    if (is.fn(options)) {
+  get(options?: GetBucketRequest): Promise<GetBucketResponse>;
+  get(callback: GetBucketCallback);
+  get(options: GetBucketRequest, callback: GetBucketCallback);
+  get(options?: GetBucketRequest|GetBucketCallback,
```

Comment:
Ah I see - Just to make this crystal clear, please rename `options` to `optionsOrCallback`.  Then create a new variable for `options` in the method.